test/functionality/dataflow/environments/resolve-tests.ts

```diff
@@ -42,4 +44,28 @@ describe('Resolve', () => {
 			});
 		});
 	});
+	describe('Builtin Constants', () => {
+		it('Unknown Identifier', () => {
+			const env = defaultEnv();
+			const result = resolvesToBuiltInConstant(undefined, env, undefined);
+			expect(result, 'should be Ternary.Never').to.be.equal(Ternary.Never);
```

Comment:
So in general we would prefer to compact this here. (e.g., avoid the `env` variable).
Besides, we only rarely use BDD here and usually use the normal assertion style.

Reply:
I used the style that was used in the file. How would the preferred style look like?

Reply:
`assert.strictEqual` etc :D